src/components/Stake/GMXAprTooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import { Trans } from "@lingui/macro";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import { formatKeyAmount } from "lib/numbers";
+
+export default function GMXAprTooltip({ processedData, nativeTokenSymbol }) {
```

Comment:
The `processedData` structure does not look very clear, I think it would be good to add `type Props` in this component 🙏